app/mailers/jobapps_mailer.rb

```diff
@@ -42,6 +42,16 @@ def interview_reschedule(interview)
          reply_to: reply_to
   end
 
+  def send_note_for_later(application_record)
```

Comment:
What about the `note_for_later`?  Aren't we sending that along with the email if it exists?